src/main/java/com/simpletalk/enterprise/dao/PostDAOStub.java

```diff
@@ -8,51 +8,31 @@
 import java.util.List;
 import java.util.Map;
 
-/**
- * <p>
- *     public class that implements methods from the IPostDAO interface
- * </p>
- */
```

Comment:
Public classes and public methods should have JavaDoc.  I'd fix this instead of remove it.